### PR TITLE
Implement ibc tx index page ui and wire up

### LINKED ISSUE #1209: Implement IBC Tx Index Page UI and wire up

### PR DESCRIPTION
**Pull request by prin-r**
Fix #1209

### CODE REVIEW

File: scan/src/subscriptions/TxSub.re [hunk 1]
```diff
@@ -288,247 +292,244 @@ module Msg = {
...
-      sender: Address.t,
-      data: string,
+    type common_t = {
       sequence: int,
       sourcePort: string,
       sourceChannel: string,
       destinationPort: string,
       destinationChannel: string,
-      timeoutHeight: int,
+      timeoutHeight: ID.Block.t,
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: It should be int.

File: scan/src/subscriptions/TxSub.re [hunk 2]
```diff
@@ -288,247 +292,244 @@ module Msg = {
...
-        destinationChannel: json |> at(["packet", "destination_channel"], string),
-        timeoutHeight: json |> at(["packet", "timeout_height"], int),
-        chainID: "band-consumer",
+        common: {
+          sequence: json |> at(["packet", "sequence"], int),
+          sourcePort: json |> at(["packet", "source_port"], string),
+          sourceChannel: json |> at(["packet", "source_channel"], string),
+          destinationPort: json |> at(["packet", "destination_port"], string),
+          destinationChannel: json |> at(["packet", "destination_channel"], string),
+          timeoutHeight: json |> at(["packet", "timeout_height"], ID.Block.fromJson),
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: decode to int

File: scan/src/subscriptions/TxSub.re [hunk 3]
```diff
@@ -288,247 +292,244 @@ module Msg = {
...
     };
     let decode = json =>
       JsonUtils.Decode.{
+        common: {
+          sequence: 999,
+          sourcePort: "gjdojfpjfp",
+          sourceChannel: "gjdojfpjfp",
+          destinationPort: "gjdojfpjfp",
+          destinationChannel: "gjdojfpjfp",
+          timeoutHeight: ID.Block.ID(999999),
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: int too

File: scan/src/subscriptions/TxSub.re [hunk 4]
```diff
@@ -288,247 +292,244 @@ module Msg = {
...
-        destinationChannel: json |> at(["packet", "destination_channel"], string),
-        timeoutHeight: json |> at(["packet", "timeout_height"], int),
-        chainID: "band-consumer",
+        common: {
+          sequence: json |> at(["packet", "sequence"], int),
+          sourcePort: json |> at(["packet", "source_port"], string),
+          sourceChannel: json |> at(["packet", "source_channel"], string),
+          destinationPort: json |> at(["packet", "destination_port"], string),
+          destinationChannel: json |> at(["packet", "destination_channel"], string),
+          timeoutHeight: json |> at(["packet", "timeout_height"], ID.Block.fromJson),
```
Comment (evilpeach) [inline scan/src/subscriptions/TxSub.re]: decode to int as well

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by tansawit — APPROVED** [review]

**Pull request merged**